### ISSUE TITLE
minor improvements to `mprof run --help`

### ISSUE THREAD

**Issue by anntzer** [fabianp/memory_profiler#125]
```
$ mprof run -h
Usage: mprof [options]

Options:
  --version             show program's version number and exit
  -h, --help            show this help message and exit
  --python              Activates extra features when the profiled executable
                        is                       a Python program (currently:
                        function timestamping.)
  --nopython            Disables extra features when the profiled executable
                        is                       a Python program (currently:
                        function timestamping.)
  -T INTERVAL, --interval=INTERVAL
                        Sampling period (in seconds), defaults to 0.1
  -C, --include-children
                        Monitors forked processes as well (sum up all process
                        memory)
```

It may be helpful to have the first line say `Usage: mprof run [options]` (at first I was a bit confused because I thought these were the global options, not the options specific to `mprof run`).
Also, note the extra spaces in the help for `--python` and `--nopython`.

**Comment by fabianp** [comment]
I agree. Can you send a patch for this?

**Comment by anntzer** [comment]
Not right now... other stuff on my plate :-)

**Comment by fabianp** [comment]
OK, fixed in 861f531 . Thanks for reporting!
